client/packages/common/src/ui/components/navigation/Tabs/DetailTabs.tsx

```diff
@@ -89,6 +91,8 @@ export const DetailTabs: FC<DetailTabsProps> = ({
         };
       });
     }
+    // isValidTab is recreated on every render, so it's safe to ignore it
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Should `isValidTab` be wrapped into a useCallback() instead?

Reply:
yes, that's more correct. have done thanks.